sys/vm/vm_page.h

```diff
@@ -290,11 +290,22 @@ struct vm_page {
  * 	 object/vm_page_t because there is no knowledge of their pte
  * 	 mappings, and such pages are also not on any PQ queue.
  *
+ * VPO_FASTCAPDIRTY is set if this page is permitted to bear capabilities, the
+ * underlying architectural permission mechanism is being used to track
+ * capability stores (as per PGA_CAPDIRTY), and the VM is aware of capabilities
+ * on this page.  In general, if PGA_CAPDIRTY is set, so will be
+ * VPO_FASTCAPDIRTY.  If both VPO_FASTCAPDIRTY and PGA_CAPDIRTY are clear, the
+ * pmap must ensure that capability stores cause traps visible to the VM so
+ * that it may begin tracking capability presence.  (This will be more relevant
+ * once the notion of Capability Load Generations land; at the moment,
+ * VPO_FASTCAPDIRTY is expected to reflect the VM object-level permission;
+ * see OBJ_NOSTORETAGS.)
```

Comment:
This comment makes it seem like FASTCAPDIRTY is corresponds to MCC (the page is permitted to contain capabilities and in fact is known to maybe contain capabilities), but the code seems to be using FASTCAPDIRTY like ASC (the page is permitted to store capabilities) - e.g. in init_pte_prot.